README.md

```diff
@@ -31,6 +31,126 @@ A [tutorial](https://github.com/bids-standard/bids-matlab/blob/master/examples/t
 
 Starting point was `spm_BIDS.m` from [SPM12](https://github.com/spm/spm12) ([documentation](https://en.wikibooks.org/wiki/SPM/BIDS#BIDS_parser_and_queries)) reformatted in a `+bids` package with dependencies to other SPM functions removed.
 
-## Other tools (MATLAB only)
-- [dicm2nii](https://github.com/xiangruili/dicm2nii): A DICOM to BIDS converter
-- [imtool3D_BIDS](https://github.com/tanguyduval/imtool3D_td): A 3D viewer for BIDS directory
+### Technical representation of the BIDS layout
+
+The BIDS layout is represented as an object/structure with the following fields:
+
+```
+BIDS.description  % from dataset_description.json
+BIDS.participants % from participants.tsv
+BIDS.sessions     % from sessions.tsv, array of Nsubjects x 1
+BIDS.scans        % from scans.tsv, array of (Nsubjects*Nsessions) x 1
+BIDS.subses       % data for each modality in each subject and (optionally) sesssion
+```
+
+The field `participants` represents the participants.tsv details.
+
+The field `sessions` has (conceptually) the same size as `participants`, although differently represented. It represents the session.tsv details for each subject.
+
+The field `scans` has the same size as `subses`. It represents the scans.tsv details for each subject and session.
+
+The field `subses` represents the data files in all subjects _and_ optionally in all sessions. Every `subses` has the modalities (anat, func, eeg, etc) as structure arrays, where each array corresponds to one data file. For example
+
+```
+>> BIDS.aubses(1)
+ans =
+  struct with fields:
+
+       name: 'sub-01'
+       path: '/Users/roboos/tmp/bids-examples/synthetic/sub-01/ses-01'
+    session: 'ses-01'
+       anat: [1×1 struct]
+       func: [8×1 struct]
+       fmap: [0×0 struct]
+        beh: [0×0 struct]
+        dwi: [0×0 struct]
+        eeg: [0×0 struct]
+        meg: [0×0 struct]
+       ieeg: [0×0 struct]
+        pet: [0×0 struct]
+```
+
+### User interface
+
+Users are not meant to directly interface with the BIDS layout object/structure, but rather use the `query` function. In general that works like
+
+```
+BIDS.query(layout, 'item') returns a complete list of the specific items
```

Comment:
The use of `BIDS` here seems inconsistent from the `BIDS.description` etc. used above. There, `BIDS` is a variable holding the BIDS layout structure. Here, `BIDS` looks like it's referencing the `+bids` package, but using the wrong case. (This may be an issue on systems with case-sensitive filesystems, like Linux.) I suggest using the `BIDS` identifier in the same way everywhere, instead of switching between using `BIDS` above and `layout` here to refer to the object/structure returned from `bids.layout(...)`.